website/content/docs/k8s/helm.mdx

```diff
@@ -766,11 +766,14 @@ Use these links to navigate to a particular top-level stanza.
 
   - `experiments` ((#v-global-experiments)) (`array<string>: []`) - Consul feature flags that will be enabled across components.
     Supported feature flags:
+    - `v1dns`:
+      When this flag is set, Consul agents use the legacy DNS implementation.
+      This setting exists in the case a DNS bug is found after the refactoring introduced in v1.19.0.
 
     Example:
 
     ```yaml
-    experiments: [ "<experiment name>" ]
+    experiments: [ "v1dns" ]
```

Comment:
```suggestion
   experiments: [ "<experiment name>" ]
```

Reply:
This is what is currently in main and release/1.19.x and has not changed recently.

Reply:
> This is what is currently in main and release/1.19.x and has not changed recently.

This is autogenerated from consul-k8s doc comments from 1.5.x to correspond to this 1.19.x branch, and this is the link to the existing consul-k8s comment this is generated from: [main](https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/values.yaml#L911-L919) and [1.5.x](https://github.com/hashicorp/consul-k8s/blob/release/1.5.x/charts/consul/values.yaml#L910-L918) so I think someone explicitly made this into an example with "v1dns" since it looks like it's the only supported value? Do you think this comment should be updated in K8s?

Reply:
👍 no this is fine.  resolving